### ISSUE TITLE
Different accuracy score comparing to DataParallel, and horovod is always better. What are the possible reasons?

### ISSUE THREAD

**Issue by Fangyh09** [horovod/horovod#1621]
Horovod version: 0.15.2
PyTorch version: 1.0.0
Hello, these are my two configurations.
1. horovod + 4gpus + total batch size 32 + real lr 0.0001 + float32
2. dataparallel + 4gpus + total batch size 32 + real lr 0.0001 / 4 + float32
( Because dataparallel sums all gradients, so I use 0.0001/4 )
Configuration 1 always gave better accuracy scores than configuration 2, what are the possible reasons?

The reasons I came up with are:
1. horovod has less numerical errors.
2. no sync batchnorm is a better thing for my task.

**Comment by tgaddair** [comment]
Hey @Fangyh09, it's tough to say without seeing the code.  Are you using local batch norm with Horovod and sync batch norm with DataParallel?  Maybe you could try removing sync batch norm and see how that performs.

**Comment by Fangyh09** [comment]
Yes, you are right. I added sync batchnorm for both. But I found recently that horovod did not support sync batchnorm.
